src/DotNetty.Common/Utilities/ITimeout.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DotNetty.Common.Utilities
+{
+    /// <summary>
+    /// A handle associated with a <see cref="ITimerTask"/> that is returned by a
+    /// <see cref="ITimer"/>.
+    /// </summary>
+    public interface ITimeout
+    {
+        /// <summary>
+        /// Returns the <see cref="ITimer"/> that created this handle.
+        /// </summary>
+        ITimer Timer { get; }
+
+        /// <summary>
+        /// Returns the <see cref="ITimerTask"/> which is associated with this handle.
+        /// </summary>
+        ITimerTask Task { get; }
+
+        /// <summary>
+        /// Returns {@code true} if and only if the <see cref="ITimerTask"/> associated
+        /// with this handle has been expired.
+        /// </summary>
+        bool Expired { get; }
+
+        /// <summary>
+        /// Returns <c>true</c> if and only if the <see cref="ITimerTask"/> associated
+        /// with this handle has been cancelled.
+        /// </summary>
+        bool Cancelled { get; }
```

Comment:
Porting down to the last typo :)

---
In reply to: [114472566](https://github.com/Azure/DotNetty/pull/242#discussion_r114472566) [](ancestors = 114472566)